.github/workflows/deploy.yml

```diff
@@ -51,9 +51,7 @@ jobs:
           PLUGIN_OWNER=$(echo "${PLUGIN_FULL_NAME}" | cut -d '/' -f 2)
           PLUGIN_NAME=$(echo "${PLUGIN_FULL_NAME}" | cut -d '/' -f 3-)
           PLUGIN_VERSION=$(yq '.plugin_version' ${PLUGIN_YAML})
-          docker pull ghcr.io/bufbuild/plugins-${PLUGIN_OWNER}-${PLUGIN_NAME}:${PLUGIN_VERSION} || :
+          docker pull ghcr.io/bufbuild/plugins-${PLUGIN_OWNER}-${PLUGIN_NAME}:${PLUGIN_VERSION}
```

Comment:
If we can't pull an image (either it doesn't exist or some other error), won't this fail with exit code 1?

Reply:
Yes - that's what we want. If we can't pull the image the `--image` flag won't work below.

Reply:
Okay perfect, just confirming this is the intended behavior.